backend/src/db/migrations/20250815022242_identity-lockouts.ts

```diff
@@ -0,0 +1,51 @@
+import { Knex } from "knex";
+
+import { TableName } from "../schemas";
+
+export async function up(knex: Knex): Promise<void> {
+  if (await knex.schema.hasTable(TableName.IdentityUniversalAuth)) {
+    const hasLockoutEnabled = await knex.schema.hasColumn(TableName.IdentityUniversalAuth, "lockoutEnabled");
```

Comment:
I think if it's integer better signify the time type as well in column. 

Like lockoutDurationInSeconds